docs/tctl/admin/history_host/index.md

```diff
@@ -0,0 +1,25 @@
+---
+id: index
+title: tctl admin history_host
+description: Overview of history_host commands
+tags:
+  - reference
+  - tctl
+---
+
+::: NOTE
+
+This documentation reflects tctl version 1.16.
+
+:::
+
```

Comment:
```suggestion

```

As long as this is on the index page (/docs/tctl/index) - then we don't need it on each command page.